dash/backend/src/email-templates/falco-log-email.hbs

```diff
@@ -0,0 +1,37 @@
+<style>
+    td{
+        text-align: left;
+        width: 80%;
+    }
+    th {
+        text-align: left;
+        width: 20%;
+    }
+</style>
+
```

Comment:
Checkout how some other apps format their emails - https://raygun.com/blog/new-error-notification-emails-packed-with-data/

Lets make a few changes here to make the email a bit nicer looking: 

1) Enclose it in a grey box 

2) At an intro before displaying message details

Hello there,

A new {Rule} error occurred in {namespace}. 

Put the contents of the falco message on ints own line. 


        <tr>
            <th colspan=2>Message</th>
        </tr>
        <tr>
            <td colspan=2>{{falcoLog.message}}</td>
        </tr>